crates/oxide/src/extractor/pre_processors/clojure.rs

```diff
@@ -5,6 +5,14 @@ use bstr::ByteSlice;
 #[derive(Debug, Default)]
 pub struct Clojure;
 
+#[inline]
+fn is_keyword_terminator(byte: u8) -> bool {
+    matches!(
+        byte,
+        b'"' | b';' | b'@' | b'^' | b'`' | b'~' | b'(' | b')' | b'[' | b']' | b'{' | b'}' | b'\\'
+    ) || byte.is_ascii_whitespace()
+}
```

Comment:
```suggestion
#[inline]
fn is_keyword_character(byte: u8) -> bool {
    return matches!(
        byte,
        b'+' | b'-' | b'/' | b'*' | b'_' | b'#' | b'.' | b':' | b'?'
    ) | byte.is_ascii_alphanumeric();
}
```

I'd go for a positive check versus a negative one. Using `|` instead of `||` produces better, branchless assembly too.

Reply:
Alright, but this excludes a lot of characters that are valid in keywords. Are you explicitly selecting the ones valid in Tailwind?

Reply:
Btw, the assembly inspector is cool!

Reply:
> Alright, but this excludes a lot of characters that are valid in keywords. Are you explicitly selecting the ones valid in Tailwind?

Kinda, yes. I tried finding a grammar for ClojureScript but I can't find any single one that "agree"s on what characters are actually allowed in a keyword. Our grammar for what's allowed in a utility name is fairly restricted *until* you open up an arbitrary value and `[]` aren't allowed in keywords. Probably need to add `%` to this list though if that's a valid keyword char.

Is there a good source that can detail what's actually allowed?

> Btw, the assembly inspector is cool!

Compiler explorer is so useful — such an awesome tool.

Reply:
Yeah, `%` is allowed. Unfortunately, the docs don't contain an exhaustive list of allowed characters, and the authoritative list is the implementation.

I've looked at the code and verified against with Claude to get a list of valid characters other than alphanumerics. I've also evaluated the monstrosity `:!#$%&'*+-.d/:<=>?_|` locally to verify this, and it works.

```
! (33) - exclamation
# (35) - hash/pound
$ (36) - dollar
% (37) - percent
& (38) - ampersand
' (39) - single quote
* (42) - asterisk
+ (43) - plus
- (45) - hyphen/minus
. (46) - period/dot
/ (47) - forward slash
: (58) - colon
< (60) - less than
= (61) - equals
> (62) - greater than
? (63) - question mark
_ (95) - underscore
| (124) - pipe/vertical bar
```

Nevertheless, there's a point to your idea of sticking with characters valid in Tailwind, as this eliminates keywords that happen to be valid in Clojure, but are definitely not valid Tailwind. It might also be somewhat less future proof, if Tailwind happens to start using a char in the future that isn't currently being used.